src/executioners/transient_executioner.hpp

```diff
@@ -29,7 +21,14 @@ class TransientExecutioner : public Executioner
 
   void Execute() const override;
 
-  hephaestus::TimeDomainProblem * _problem{nullptr};
+private:
+  double _t_initial;       // Start time
+  double _t_final;         // End time
+  mutable double _t;       // Current time
+  mutable int _it;         // Time index
+  int _vis_steps;          // Number of cyces between each output update
+  mutable bool _last_step; // Flag to check if current step is final
+  TimeDomainProblem * _problem{nullptr};
```

Comment:
Similarly to hephaestus::SteadyStateProblem, minor preference for being explicit here (hephaestus::TimeDomainProblem *)